client/components/global/ChangePasswordForm.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useContext } from 'react';
+import { BrowserRouter as Router, Switch, Route, Link, useParams } from 'react-router-dom';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import axios from 'axios';
+
+export const ChangePasswordForm = () => {
+  let user = useParams();
+  const [show, setShow] = useState(false);
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+  const [err, setErr] = useState(null);
+  const [noMatch, setNoMatch] = useState(false);
+  const [response, setResponse] = useState(null);
+
+  const handleChangePassword = () => {
```

Comment:
May be due to the overall complexity of this component, but not a huge fan of having the axios helper live inside (I see that it needs access to a lot of pieces, so it may take some thought-work if you wanted to refactor that).  I think it will be okay with some of the changes mentioned below regarding the other section of the component